api/api.go

```diff
@@ -286,6 +302,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 
 		status := api.getAlertStatus(a.Fingerprint())
 
+		if !showSilenced && len(status.SilencedBy) != 0 {
```

Comment:
Yea, practically, one probably just wants to hide everything non-firing. But lets stick to silencing only for now. Few people are using inhibition anyway.

We should definitely have an inhibition indicator in the UI for feature parity.